lib/PrefixedSparqlClient.js

```diff
@@ -0,0 +1,10 @@
+function createPrefixedSparqlClient ({ client, queryPrefix }) {
+  const prefix = queryPrefix
+  return {
+    query: {
+      construct: (query, ...args) => client.query.construct(`${prefix}\n${query}`, ...args),
+      select: (query, ...args) => client.query.select(`${prefix}\n${query}`, ...args)
+    }
+  }
+}
```

Comment:
Wait, that is not enough. The client implements a specific interface for the [query class](https://github.com/bergos/sparql-http-client/blob/master/StreamQuery.js#L10-L92). It may cause problems if you simply reduce that interface to `construct/select` :)

Maybe this calls for inheritance?

Reply:
I think this violates the principle of least surprise. I understand that only these two methods are used in this library but the assumption is that the sparql client will be compatible with the base `sparql-http-client`

What do you think @bergos ?